model/delivery.py

```diff
@@ -101,19 +101,25 @@ def scope(self):
 
 
 class SigningCfg(ModelBase):
+    def id(self) -> str:
+        return self.raw.get('id')
+
     def algorithm(self):
         return self.raw.get('algorithm', 'HS256')
 
     def secret(self):
         return self.raw.get('secret')
 
+    def purpose_label(self):
```

Comment:
maybe it would be more "future-proof" to make this a `list` (label_s_), even if we will only have one single label to start with?